lib/Cpdf.php

```diff
@@ -3198,6 +3418,12 @@ function output($debug = false)
 
         $xref = [];
         $content = '%PDF-' . self::PDF_VERSION;
+
+        if ($this->pdfa) {
+            // Force binary mode with 4 random bytes above 127
+            $content .= "\n%" . chr(rand(128, 256)) . chr(rand(128, 256)) . chr(rand(128, 256)) . chr(rand(128, 256));
```

Comment:
FYI, the max value for the `rand` function should be 255 since, per documentation, the `chr` codepoint should be "An integer between 0 and 255." The function `$bytevalue %= 256;`'s the value, which results in a codepoint of 0 when 256 is passed in.

Noting for reference, will fix in an upcoming push.